kernel/comps/virtio/src/transport/mod.rs

```diff
@@ -24,16 +24,17 @@ pub mod pci;
 pub trait VirtioTransport: Sync + Send + Debug {
     // ====================Device related APIs=======================
 
-    fn device_type(&self) -> VirtioDeviceType;
+    /// Get device type.
+    fn get_device_type(&self) -> VirtioDeviceType;
```

Comment:
The Rust naming convention is that for simple getter methods, skip the `get_` prefix. So this method name should not be changed.